homeassistant/components/zwave_me/const.py

```diff
@@ -0,0 +1,31 @@
+"""Constants for ZWaveMe."""
+# Base component constants
+NAME = "Z-Wave-Me"
+DOMAIN = "zwave_me"
+DOMAIN_DATA = f"{DOMAIN}_data"
+VERSION = "0.0.1"
+ATTRIBUTION = "Z-Wave-Me"
+ISSUE_URL = "https://github.com/Z-Wave-Me/ha-core/pulls"
```

Comment:
We want users to report issues first to the core repository and then we'll redirect them if needed after triage. You will be notified of issues as codeowners when the issues are labeled with the integration label.

Reply:
Shall we change it to https://github.com/home-assistant/core/issues ?

Reply:
I think we can remove this constant string. It's not used anywhere.